source/_posts/2018/2018-10-08-new-in-symplify-5-create-merge-and-split-monorepo-with-1-command.md

```diff
@@ -115,4 +115,4 @@ and read [`README`](https://github.com/symplify/monorepobuilder) for detail usag
 
 <br>
 
-Happy monorapping!
+Happy monorepping!
```

Comment:
This one is intended ;)

Reply:
Ah, I wasn't quite sure about this one :)

Reply:
I like to play with words... There is also 3rd typo possible in this word ;)